File: src/main/java/eterea/api/rest/repository/IArticuloMovimientoRepository.java
/**
 * 
 */
package eterea.api.rest.repository;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import eterea.api.rest.model.ArticuloMovimiento;

/**
 * @author daniel
 *
 */
@Repository
public interface IArticuloMovimientoRepository extends JpaRepository<ArticuloMovimiento, Long> {

	public List<ArticuloMovimiento> findAllByClienteMovimientoId(Long clienteMovimientoId);

}

File: src/main/java/eterea/core/api/rest/kotlin/repository/IArticuloMovimientoRepository.kt
package eterea.core.api.rest.kotlin.repository

import eterea.core.api.rest.kotlin.model.ArticuloMovimiento
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.Optional

@Repository
interface IArticuloMovimientoRepository : JpaRepository<ArticuloMovimiento?, Long?> {
    fun findAllByClienteMovimientoId(clienteMovimientoId: Long?): List<ArticuloMovimiento?>?

    fun findByArticuloMovimientoId(articuloMovimientoId: Long): Optional<ArticuloMovimiento?>?

}
